src/main/scala/com/raquo/airstream/combine/generated/StaticEventStreamCombineOps.scala

```diff
@@ -0,0 +1,148 @@
+package com.raquo.airstream.combine.generated
+
+import com.raquo.airstream.core.EventStream
+
+// #Warning do not edit this file directly, it is generated by GenerateStaticEventStreamCombineOps.scala
+
+// These combine and combineWith methods are available on the EventStream companion object
+// For instance methods of the same name, see CombinableEventStream.scala
+
+object StaticEventStreamCombineOps {
+
+  def combine[T1, T2](
```

Comment:
Then you could use `zip` for here as well :D I see why combineWith may be a lil' strange to have as a companion function.